cegs_portal/search/tsv_templates/v1/reg_effects.py

```diff
@@ -1,23 +1,110 @@
+                    source.location.upper,
+                    tested_element(source, target),
+                    "0",
+                    if_strand(source.strand),
+                ]
+
+                tsv_data.append(row)
+
+    return tsv_data
+
+
+def reg_effects(reos, options):
+    if is_bed6(options):
+        return bed6_output(reos)
     tsv_data = []
     tsv_data.append(
-        ["Accession ID", "Effect Size", "Direction", "Significance", "Distance", "Feature Type", "Source", "Experiment"]
+        [
+            "#chrom",
+            "chromStart",
+            "chromEnd",
+            "name",
+            "score",
+            "strand",
+            "Effect Size (log2FC)",
+            "Direction",
+            "Significance",
+            "Experiment",
+            "Target",
+        ]
     )
 
-    for reo in reg_effects_page:
-        first_source = reo.sources.all()[0]
-        row = [
-            reo.accession_id,
-            reo.effect_size,
-            reo.direction,
-            reo.significance,
-            first_source.closest_gene_distance,
-            first_source.get_feature_type_display(),
-            first_source.accession_id,
-            reo.experiment.name,
+    for reo in reos:
+        for target in reo.targets.all():
+            for source in reo.sources.all():
+                row = [
+                    source.chrom_name,
+                    source.location.lower,
+                    source.location.upper,
+                    tested_element(source, target),
+                    "0",
+                    if_strand(source.strand),
+                    reo.effect_size,
+                    reo.direction,
+                    format_float(reo.significance),
+                    reo.experiment.name,
+                    target.name,
+                ]
+
+                tsv_data.append(row)
+
+    return tsv_data
+
+
+def target_reg_effects(reos, options):
+    if is_bed6(options):
+        return bed6_output(reos)
+    tsv_data = []
+    tsv_data.append(
+        [
+            "#chrom",
+            "chromStart",
+            "chromEnd",
+            "name",
+            "score",
+            "strand",
+            "Tested Elements",
+            "Effect Size",
+            "Direction",
+            "Significance",
+            "Experiment",
         ]
+    )
+
+    for reo in reos:
+        for source in reo.sources.all():
+            for target in reo.targets.all():
+                row = [
+                    source.chrom_name,
+                    source.location.lower,
+                    source.location.upper,
+                    tested_element(source, target),
+                    "0",
+                    if_strand(source.strand),
+                    reo.effect_size,
+                    reo.direction,
+                    format_float(reo.significance),
+                    reo.experiment.name,
+                ]
 
-        tsv_data.append(row)
+            tsv_data.append(row)
```

Comment:
this needs to be indented one more unit to the left, so it's at the same level as `row = [`